src/store/storage-controller.ts

```diff
@@ -78,6 +80,9 @@ export class StorageController {
     }
 
     await messageStore.put(tenant, message, indexes);
+
+    const messageCid = await Message.getCid(message);
```

Comment:
we already calculate this above, but only `if (message.descriptor.dataCid !== undefined)`

could we instead move that outside the `if` and avoid the repeated calculation?